x/qgb/abci.go

```diff
@@ -9,6 +9,18 @@ import (
 
 // EndBlocker is called at the end of every block
 func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
+	handleValset(ctx, k)
+	handleDataCommitment(ctx, k)
```

Comment:
similar nit as above, I think these should be named a bit differently. while this works fine, they are creating data commitment and valset requests, not really handling a valset or data commitment